src/Nox.Generator/Application/Commands/PartialUpdateOwnedCommandGenerator.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.CodeAnalysis;
+using Nox.Generator.Common;
+using Nox.Solution;
+using System.Linq;
+
+namespace Nox.Generator.Application.Commands;
+
+internal class PartialUpdateOwnedCommandGenerator : INoxCodeGenerator
+{
+    public NoxGeneratorKind GeneratorKind => NoxGeneratorKind.Domain;
+
+    public void Generate(SourceProductionContext context, NoxSolutionCodeGeneratorState codeGeneratorState, GeneratorConfig config)
+    {
+        context.CancellationToken.ThrowIfCancellationRequested();
+
+        if (codeGeneratorState.Solution.Domain is null)
+        {
+            return;
+        }
+
+        var templateName = @"Application.Commands.PartialUpdateOwnedCommand";
+
+        foreach (var entity in codeGeneratorState.Solution.Domain.Entities.Where(x => x.IsOwnedEntity))
+        {
+            context.CancellationToken.ThrowIfCancellationRequested();
+
+            var parent = codeGeneratorState.Solution.Domain.Entities.FirstOrDefault(e =>
+                e.OwnedRelationships?.Any(o => o.Entity == entity.Name && !o.WithSingleEntity) == true);
```

Comment:
Do we need "== true"?

Reply:
yes, because of the null-conditional operator for `e.OwnedRelationships?`